test/configCases/css/local-ident-keywords/style.module.css

```
@@ -0,0 +1,30 @@
+/* A keyword in a value position that also accepts a `<custom-ident>` must stay
+   global — scoping one renames something the UA defines. */
+.animation-keywords {
+	animation-name: none;
+	animation: 1s linear infinite alternate both running paused;
```

Stylelint (17.14.0)

[error] Overridden property "animation-name" by shorthand "animation" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)